src/test-collection/formatters/list.ts

```diff
@@ -0,0 +1,33 @@
+import path from "node:path";
+import type { TestCollection, TestDisabled, FormatterListTest } from "..";
+
+export const format = (testCollection: TestCollection): FormatterListTest[] => {
+    const allTestsById = new Map<string, FormatterListTest>();
+
+    testCollection.eachTest((test, browserId) => {
+        if ((test as TestDisabled).disabled) {
+            return;
+        }
+
+        if (allTestsById.has(test.id)) {
+            const foundTest = allTestsById.get(test.id)!;
```

Comment:
for the same test in different browsers, I specify only the field `browserIds`. In order not to store too much of the same data (except for the browser name)